workflows/genome_annotation/annotation_helixer/README.md

```diff
@@ -0,0 +1,64 @@
+
+
+Helixer is an annotation software with a new and different approach: it performs evidence-free predictions (no need for RNASeq data or sequence aligments), using Graphics Processing Unit (GPU), with a much faster execution time. The annotation is based on the development and use of a cross-species deep learning model. The software is used to configure and train models for ab initio prediction of gene structure. In other words, it identifies the base pairs in a genome that belong to the UTR/CDS/Intron genes.
+
+To assess the quality of the proteome, we will use the GFFRead tool to extract the predicted protein sequences from the annotation (i.e. the Helixer annotation).
+
+To assess the quality of the annotation, we will use different tools:
+- Genome Annotation Statistics: is a program designed to analyze and provide statistics on genomic annotations. This software performs its analyses from a GFF3 file.
+- BUSCO (Benchmarking Universal Single-Copy Orthologs):  is a tool allowing to evaluate the quality of a genome assembly or of a genome annotation. By comparing genomes from various more or less related species, the authors determined sets of ortholog genes that are present in single copy in (almost) all the species of a clade (Bacteria, Fungi, Plants, Insects, Mammals, …). Most of these genes are essential for the organism to live, and are expected to be found in any newly sequenced and annotated genome from the corresponding clade. Using this data, BUSCO is able to evaluate the proportion of these essential genes (also named BUSCOs) found in a set of (predicted) transcript or protein sequences. This is a good evaluation of the “completeness” of the annotation.
+- OMArk: is proteome quality assessment software. It provides measures of proteome completeness, characterises the consistency of all protein-coding genes with their homologues and identifies the presence of contamination by other species. OMArk is based on the OMA orthology database, from which it exploits orthology relationships, and on the OMAmer software for rapid placement of all proteins in gene families.
+
+The final step is to view the generated annotation using a genome browser such as JBrowse. This browser allows you to navigate along the chromosomes of the genome and view the structure of each predicted gene.
+
+## Input dataset for Helixer
+Helixer requires the genome sequence to be annotated, in fasta format.
+
+## Output dataset for Helixer
+Helixer produces a single output dataset: a GFF3 file. The GFF3 format is a standard bioinformatics format for storing genome annotations. Each row describes a genomic entity, with columns detailing its identifier, location, score and other attributes.
+
+## Input dataset for Genome Annotation Statistics
```

Comment:
It's better to just describe the workflow inputs, you don't need to describe all the inputs to intermediate steps. The idea is that the user knows what they have to provide as the input, and what files they can expect as outputs.